src/controllers/itemdetailpage.js

```diff
@@ -406,9 +405,18 @@ define(["loading", "appRouter", "layoutManager", "connectionManager", "userSetti
 
         if (parentNameHtml.length) {
             if (parentNameLast) {
-                html = '<h3 class="parentName" style="margin: .25em 0;">' + parentNameHtml.join("</br>") + "</h3>";
+                // Music
+                if (layoutManager.mobile) {
+                    html = '<h3 class="parentName" style="margin: .25em 0;">' + parentNameHtml.join("</br>") + "</h3>";
+                } else {
+                    html = '<h3 class="parentName" style="margin: .25em 0;">' + parentNameHtml.join(" - ") + "</h3>";
+                }
             } else {
-                html = '<h1 class="parentName" style="margin: .1em 0 .25em;">' + parentNameHtml.join("</br>") + "</h1>";
+                if (layoutManager.mobile) {
+                    html = '<h1 class="parentName" style="margin: .1em 0 .25em;">' + parentNameHtml.join("</br>") + "</h1>";
+                } else {
+                    html = '<h1 class="parentName" style="margin: .1em 0 .25em;">' + parentNameHtml[0] + "</h1>";
```

Comment:
why use only `parentNameHtml[0]` on desktop but all (joined by `<br>`) on mobile? doesn't make much sense IMHO

Reply:
@JustAMan That's what @MrTimscampi proposed earlier: Keep Show, season and episode name in different lines for mobile, while keeping TV Show in one line and Episode and Season in the same line but in a different one.

``parentNameHTML`` array contains the html code for each of the elements that are parent of an specific one. I tested it in all kinds of elements and it can only have two entries, no more, so 0 and 1 should be always indexes inside the ranges (unless I missed something).

``0`` is the TV Show. ``1`` is the season. See the GIF I attached for reference.

And yes, probably I can remove the ``<br>``, as I added the flex container afterwards, which should handle the spacing. Let me verify that.

Reply:
It'd probably be nicer if parentNameHTML was divided into variables with a name related to their content. It makes it easier to read and understand what's going on.

Reply:
@JustAMan Now that I tested I know why I used ``<br>``. Flex already does the job correctly on mobile devices, but not in tablets. See iPad emulator in Firefox:

**Without**
![image](https://user-images.githubusercontent.com/10274099/78478439-46a6b280-7745-11ea-90b4-3e914257b39c.png)

**With**
![image](https://user-images.githubusercontent.com/10274099/78478815-6047fa00-7745-11ea-8be6-2702f09c609f.png)

Take in mind that the array doesn't include ``Como conocí a vuestra madre`` or ``Temporada 1``. It includes ``<a style="color:inherit;" class="button-link emby-button" is="emby-linkbutton" href="itemdetails.html?id=bb0ec83....&serverId=606da8b....">Cómo conocí a vuestra madre</a>`` and ``<a style="color:inherit;" class="button-link emby-button" is="emby-linkbutton" href="itemdetails.html?id=173ad87.....&serverId=606da8b77....">Temporada 1</a>`` which are generated before. That's why I say that this need a huge refactoring, because it's a mess :)

Reply:
> Take in mind that the array doesn't include Como conocí a vuestra madre or Temporada 1. It includes <a style="color:inherit;" class="button-link emby-button" is="emby-linkbutton" href="itemdetails.html?id=bb0ec83....&serverId=606da8b....">Cómo conocí a vuestra madre</a> and <a style="color:inherit;" class="button-link emby-button" is="emby-linkbutton" href="itemdetails.html?id=173ad87.....&serverId=606da8b77....">Temporada 1</a> instead, which are generated before. That's why I say that this need a huge refactoring, because it's a mess :)

It's still easier if it's like:
```javascript
var showNameHtml = a style="color:inherit;" class="button-link emby-button" is="emby-linkbutton" href="itemdetails.html?id=bb0ec83....&serverId=606da8b....">Cómo conocí a vuestra madre</a>;
var seasonNameHtml = <a style="color:inherit;" class="button-link emby-button" is="emby-linkbutton" href="itemdetails.html?id=173ad87.....&serverId=606da8b77....">Temporada 1</a>;
```

It's easier to understand than having to go back up to look at .append() then inferring the position in the array to know what's in there :)

Reply:
@MrTimscampi Okay, now I understand what you mean